src/components/Account.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box, Typography } from '@mui/material';
+
+type AccountProps = {
+    name: string
+}
+
+export const Account = (props: AccountProps) => {
+    return (
+        <Box sx={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
+            <Typography>{props.name}</Typography>
+            <Box
+                component='img'
+                sx={{
+                    height: 70,
+                    width: 70,
+                    borderRadius: 25,
+                    marginLeft: 3,
```

Comment:
détail mais si tu veux être plus compact tu peux remplacer "marginLeft" par "ml". Même chose pour toute les autres directions ainsi que pour les types de padding.

Reply:
Parfait! C'est noté.